### PR TITLE
New package: frontier-client

### PR DESCRIPTION
**Pull request by iarspider**
Re-do of #21373

### CODE REVIEW

**Comment by iarspider** [comment]
@tldahlgren ping

File: var/spack/repos/builtin/packages/frontier-client/package.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+class FrontierClient(MakefilePackage):
+    """The Frontier distributed database caching system distributes data from
+       data sources to many client systems around the world. The name comes
+       from "N Tier" where N is any number and tiers are layers of locations
+       of distribution."""
+
+    homepage = "http://frontier.cern.ch/"
+    url      = "https://github.com/fermitools/frontier/archive/v2_9_1.tar.gz"
+
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/frontier-client/package.py:17]: Care to add yourself as a maintainer for this package?

**Review by tldahlgren — APPROVED** [review]
LGTM.  My only question is if you want to add yourself as the Spack package maintainer for this software.

File: var/spack/repos/builtin/packages/frontier-client/package.py (hunk 1 above)
Comment (tldahlgren) [reply var/spack/repos/builtin/packages/frontier-client/package.py:17]: Given your response to a previous PR I'll withdraw my question.

**Comment by tldahlgren** [comment]
Thank you for your contribution!

**Pull request merged**